<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTreeCtrlTimeDlg</class>
 <widget class="QDialog" name="QTreeCtrlTimeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时钟</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>时钟</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>7</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>下载系统时间到HMI</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="treectrltimedlg_timetoHMI_groupbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="title">
      <string>写时间到PLC</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="timetoplc_datastyle_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>数据类型：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="timetoplc_datastyle_lineedit">
      <property name="geometry">
       <rect>
        <x>94</x>
        <y>20</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="timetoplc_size_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>44</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>长    度：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="timetoplc_size_lineedit">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>45</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="timetoplc_writeaddress_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>写入地址：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="timetoplc_writeaddress_lineEdit">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>70</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="timetoplc_writeaddress_pushButton">
      <property name="geometry">
       <rect>
        <x>293</x>
        <y>69</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="timetoplc_startstyle_groupbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>启动方式</string>
      </property>
      <widget class="QRadioButton" name="timetoplc_timed_radiobutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>计时</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="timetoplc_trigerred_radiobutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>触发</string>
       </property>
      </widget>
      <widget class="QLabel" name="timetoplc_interval_label">
       <property name="geometry">
        <rect>
         <x>83</x>
         <y>20</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>间隔：</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="timetoplc_interval_spinBox">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>19</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="timetoplc_trigered_lineEdit">
       <property name="geometry">
        <rect>
         <x>84</x>
         <y>49</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_s">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>22</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>秒</string>
       </property>
      </widget>
      <widget class="QPushButton" name="timetoplc_trigered_pushButton">
       <property name="geometry">
        <rect>
         <x>264</x>
         <y>47</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resourse.qrc">
         <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>77</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>自动复位</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="treectrltimedlg_sametimewithPLC_groupbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>331</width>
       <height>231</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="title">
      <string>与PLC时间相同</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <widget class="QComboBox" name="sametimewithPLC_datastyle_comboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sametimewithPLC_datastyle_label">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>24</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>数据类型：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sametimewithPLC_size_comboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sametimewithPLC_size_label">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>52</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>长    度：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sametimewithPLC_readaddress_lineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sametimewithPLC_readaddress_pushButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>79</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="sametimewithPLC_startstyle_groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>启动方式</string>
      </property>
      <widget class="QRadioButton" name="sametimewithPLC_timed_radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>计时</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="sametimewithPLC_trigered_radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>触发</string>
       </property>
      </widget>
      <widget class="QLabel" name="sametimewithPLC_interval_label">
       <property name="geometry">
        <rect>
         <x>84</x>
         <y>17</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>间隔：</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="sametimewithPLC_interval_spinBox">
       <property name="geometry">
        <rect>
         <x>134</x>
         <y>17</y>
         <width>125</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="sametimewithPLC_trigered_lineEdit">
       <property name="geometry">
        <rect>
         <x>83</x>
         <y>49</y>
         <width>179</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>秒</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sametimewithPLC_trigered_pushButton">
       <property name="geometry">
        <rect>
         <x>261</x>
         <y>51</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resourse.qrc">
         <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>78</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>自动复位</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="sametimewithPLC_readaddress_label">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>80</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>读取地址：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>timetoplc_datastyle_lineedit</tabstop>
  <tabstop>timetoplc_size_lineedit</tabstop>
  <tabstop>timetoplc_writeaddress_pushButton</tabstop>
  <tabstop>timetoplc_writeaddress_lineEdit</tabstop>
  <tabstop>timetoplc_timed_radiobutton</tabstop>
  <tabstop>timetoplc_trigerred_radiobutton</tabstop>
  <tabstop>timetoplc_interval_spinBox</tabstop>
  <tabstop>timetoplc_trigered_lineEdit</tabstop>
  <tabstop>timetoplc_trigered_pushButton</tabstop>
  <tabstop>sametimewithPLC_datastyle_comboBox</tabstop>
  <tabstop>sametimewithPLC_size_comboBox</tabstop>
  <tabstop>sametimewithPLC_readaddress_lineEdit</tabstop>
  <tabstop>sametimewithPLC_readaddress_pushButton</tabstop>
  <tabstop>sametimewithPLC_timed_radioButton</tabstop>
  <tabstop>sametimewithPLC_trigered_radioButton</tabstop>
  <tabstop>sametimewithPLC_interval_spinBox</tabstop>
  <tabstop>sametimewithPLC_trigered_lineEdit</tabstop>
  <tabstop>sametimewithPLC_trigered_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
